data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -0,0 +1,30 @@
+import http.client
+import json
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+SURVEYMONKEY_ACCESS_TOKEN_ENV_VAR = "SURVEYMONKEY_ACCESS_TOKEN"
+
+
+def get_connection_surveymonkey_api() -> http:
+    return http.client.HTTPSConnection("api.surveymonkey.com")
+
+
+def get_surveymonkey_api_headers() -> dict:
+    return {
+        'Accept': "application/json",
+        'Authorization': f"Bearer {SURVEYMONKEY_ACCESS_TOKEN_ENV_VAR}"
```

Comment:
I guess here you will need to get the actual token, not the env var (or filename to the token).

Reply:
Ah yes I missed! Thank you